Frameworks/CoreText/CTLine.mm

```diff
@@ -48,16 +60,7 @@ - (instancetype)copyWithZone:(NSZone*)zone {
  @Status Stub
 */
 CTLineRef CTLineCreateWithAttributedString(CFAttributedStringRef string) {
-    UNIMPLEMENTED();
-    NSString* str = [(NSAttributedString*)string string];
-    CFRange lineRange;
-    lineRange.location = 0;
-    lineRange.length = [str length];
-
-    _CTLine* line = [_CTLine new];
-    line->_strRange = lineRange;
-
-    return (CTLineRef)line;
+    return static_cast<CTLineRef>(_DWriteGetLine(string));
```

Comment:
>_DWriteGetLine [](start = 34, length = 14)

The API says it we should return NULL on a failure but I see _DWriteGetLine is returning a new CTLine object